app/views/articles/_sidebar_campaign.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="widget">
   <% if SiteConfig.campaign_sidebar_image %>
-    <img src="<%= SiteConfig.campaign_sidebar_image %>" class="widget-image" />
+    <img src="<%= SiteConfig.campaign_sidebar_image %>" class="widget-image" style="height:116px;" />
```

Comment:
I'm guessing the inline style is temporary as the campaign will change again at some point? If not, you could just add a `widget-image--campaign` CSS class if all campaigns will have this height. Also, any reason a width isn't set?

Reply:
Width is auto...

I don't know if this will be the permanent way we do things, more a patch for the moment. I think inline here works.